scripts/bitc.py

```diff
@@ -178,7 +178,7 @@ def setup_drawtext(args, filename):
     font_size = video_height / 12
     watermark_size = video_height / 14
     if sys.platform == "darwin":
-        font_path = "fontfile=/Library/Fonts/AppleGothic.ttf"
+        font_path = "fontfile=/System/Library/Fonts/AppleSDGothicNeo.ttc"
```

Comment:
This font seems a bit problematic. I'm not sure why I picked AppleGothic back in the, but when i tested this font, the timecode box changes size all the time probably due to the width differences between characters. I did a quick test using Courier instead and the timecode box and font stays in a fixed position. I'm not married to Courier, but any font that looks good and has consistent widths are essential for this script.